apps/rehabstod/src/pages/Patient/Patient.tsx

```diff
@@ -10,7 +10,7 @@ import { PatientOverview } from './components/patientOverview/PatientOverview'
 export function Patient() {
   const { patientId } = useParams()
```

Comment:
Should this variable be renamed to something like encryptedPatientId, to better mirror the value? Im open for suggestion.

Reply:
I think encryptedPatientId makes the code more understandable

Reply:
updateded!